source/jormungandr/jormungandr/olympic_site_params_manager.py

```diff
@@ -30,19 +30,43 @@
 import logging
 import json
 from collections import namedtuple
-from jormungandr.utils import get_olympic_site, local_str_date_to_utc, date_to_timestamp, str_to_dt
+from jormungandr.utils import (
+    get_olympic_site,
+    local_str_date_to_utc,
+    date_to_timestamp,
+    str_to_dt,
+    dt_to_date_str,
+)
 import boto3
-from jormungandr import app
 from botocore.client import Config
+from jormungandr import app, memory_cache, cache
 
 
 AttractivityVirtualFallback = namedtuple("AttractivityVirtualFallback", "attractivity, virtual_duration")
 
 
+class ResourceS3Object:
+    def __init__(self, s3_object, instance_name):
+        self.s3_object = s3_object
+        self.instance_name = instance_name
+
+    def __repr__(self):
+        return "{}-{}-{}".format(
```

Comment:
you can use the `ETag` as the unique key of you object

https://docs.aws.amazon.com/AmazonS3/latest/API/API_Object.html

Reply:
Done